app/src/main/java/com/example/moviews/service/ServiceNotify.kt

```diff
@@ -0,0 +1,98 @@
+package com.example.moviews.service
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.widget.RemoteViews
+import androidx.core.app.JobIntentService
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.target.NotificationTarget
+import com.example.moviews.MainActivity
+import com.example.moviews.R
+import com.example.moviews.data.OnLoadDataCallback
+import com.example.moviews.data.model.Movie
+import com.example.moviews.repository.RepositoryUtils
+import com.example.moviews.utils.BaseUrl
+import com.example.moviews.utils.Constant
+import java.util.*
+
+class ServiceNotify : JobIntentService() {
+
+    override fun onHandleWork(intent: Intent) {
+        val movieRepository = RepositoryUtils.getMovieRepository(applicationContext)
+        movieRepository.getMovies(
+            Constant.BASE_TRENDING,
+            object : OnLoadDataCallback<MutableList<Movie>> {
+                override fun onSuccess(data: MutableList<Movie>) {
+                    val random = Random().nextInt(data.size)
+                    showNotification(data[random])
```

Comment:
Collection hỗ trợ sẵn random 
```suggestion
                    showNotification(data.random())
```